apparmor.d/groups/bus/dbus-daemon-launch-helper

```diff
@@ -22,8 +22,11 @@ profile dbus-daemon-launch-helper @{exec_path} {
   /{usr/,}lib/cups-pk-helper-mechanism                      rPx,
   /{usr/,}lib/cups-pk-helper/cups-pk-helper-mechanism       rPx,
   /{usr/,}lib/software-properties/software-properties-dbus  rPx,
+  @{libexec}/language-selector/ls-dbus-backend              rPx,
```

Comment:
Same, no profile for this. So either you add a profile or you `rPUx` as a temporary solution before adding the profile.